installing/install_config/installing-customizing.adoc

```diff
@@ -30,19 +30,24 @@ include::modules/installation-special-config-butane-create.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-kmod_installing-customizing[Adding kernel modules to nodes]
-* xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-storage_installing-customizing[Encrypting or mirroring disks during installation]
+* xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-storage_installing-customizing[Encrypting and mirroring disks during installation]
 
 include::modules/installation-special-config-kargs.adoc[leveloffset=+1]
 ifdef::openshift-webscale[]
 include::modules/installation-special-config-rtkernel.adoc[leveloffset=+1]
 endif::openshift-webscale[]
 include::modules/installation-special-config-kmod.adoc[leveloffset=+1]
 include::modules/installation-special-config-storage.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening[Configuring automated unlocking of encrypted volumes using policy-based decryption] for more information about the TPM v2 and Tang encryption modes.
```

Comment:
Not sure if we have any rule on this, but I was trained to put the _For more information_ before the _see_ link. IBM style guide uses the same format, and our docs run ~2-to-1 in the _For more information, see_ format.

Reply:
I have resolved the "For more information" suggestion and have moved the "Additional resources" section to module as suggested, because it does not contain any cross-references.